webhook/hooks/deploy.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+main() {
+    echo "Deploying..."
+
+    if [ -z "${HOOK_tag}" ]; then
+      echo "The tag is not set. Skipping deployment."
+      exit 1
+    fi
+
+    if [ -z "${HOOK_image}" ]; then
```

Comment:
I'm not sure if we should allow this.  Giving the ability to specify arbitrary images seems dangerous to me.  I think we'll only update the main web service, so we can probably remove this until it's needed.

Reply:
Resolved. You can now specify the TARGET_IMAGE and TARGET_SERVICE inside the script, and the hook only provides the tag param now.